INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/_ping HTTP/1.1" 200 2
INFO     root:conftest.py:121 Env variable TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE not set, setting it now
INFO     root:conftest.py:123 We are on Linux, setting socket_path='/run/user/16481/podman/podman.sock' for TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE
WARNING  root:conftest.py:140 Ryuk is enabled. This may not work with rootless podman.
INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
INFO     testcontainers.core.container:container.py:98 Pulling image testcontainers/ryuk:0.8.1
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/create?name=testcontainers-ryuk-b0bcab84-dda0-4e93-ab06-3f49dbae1c19 HTTP/1.1" 201 88
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/fd38a9920ac829b231a98304fec78bdadada17d3a0d468389952c3a6da9f705e/json HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/fd38a9920ac829b231a98304fec78bdadada17d3a0d468389952c3a6da9f705e/start HTTP/1.1" 204 0
INFO     testcontainers.core.container:container.py:125 Container started: fd38a9920ac8
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/fd38a9920ac829b231a98304fec78bdadada17d3a0d468389952c3a6da9f705e/logs?stderr=0&stdout=1&timestamps=0&follow=0&tail=all HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/fd38a9920ac829b231a98304fec78bdadada17d3a0d468389952c3a6da9f705e/json HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/fd38a9920ac829b231a98304fec78bdadada17d3a0d468389952c3a6da9f705e/logs?stderr=1&stdout=0&timestamps=0&follow=0&tail=all HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/fd38a9920ac829b231a98304fec78bdadada17d3a0d468389952c3a6da9f705e/json HTTP/1.1" 200 None
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: fd38a9920ac8> with image testcontainers/ryuk:0.8.1 to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/fd38a9920ac829b231a98304fec78bdadada17d3a0d468389952c3a6da9f705e/json HTTP/1.1" 200 None
INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/images/quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506/json HTTP/1.1" 200 None
INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
INFO     testcontainers.core.container:container.py:98 Pulling image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'>
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/create HTTP/1.1" 201 88
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/json HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/start HTTP/1.1" 204 0
INFO     testcontainers.core.container:container.py:125 Container started: 45bcdc1a907c
INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/json HTTP/1.1" 200 None
DEBUG    root:workbench_image_test.py:194 {'Binds': [], 'ContainerIDFile': '', 'LogConfig': {'Type': 'journald', 'Config': None}, 'NetworkMode': 'bridge', 'PortBindings': {'8888/tcp': [{'HostIp': '0.0.0.0', 'HostPort': '41335'}]}, 'RestartPolicy': {'Name': 'no', 'MaximumRetryCount': 0}, 'AutoRemove': False, 'VolumeDriver': '', 'VolumesFrom': None, 'ConsoleSize': [0, 0], 'Annotations': {'io.container.manager': 'libpod', 'org.opencontainers.image.stopSignal': '15', 'org.systemd.property.KillSignal': '15', 'org.systemd.property.TimeoutStopUSec': 'uint64 10000000'}, 'CapAdd': [], 'CapDrop': [], 'CgroupnsMode': '', 'Dns': [], 'DnsOptions': [], 'DnsSearch': [], 'ExtraHosts': [], 'GroupAdd': ['0'], 'IpcMode': 'shareable', 'Cgroup': '', 'Links': None, 'OomScoreAdj': 200, 'PidMode': 'private', 'Privileged': False, 'PublishAllPorts': False, 'ReadonlyRootfs': False, 'SecurityOpt': [], 'Tmpfs': {'/opt/app-root/src': 'rw,rprivate,nosuid,nodev'}, 'UTSMode': 'private', 'UsernsMode': '', 'ShmSize': 65536000, 'Runtime': 'oci', 'Isolation': '', 'CpuShares': 0, 'Memory': 0, 'NanoCpus': 0, 'CgroupParent': 'user.slice', 'BlkioWeight': 0, 'BlkioWeightDevice': None, 'BlkioDeviceReadBps': None, 'BlkioDeviceWriteBps': None, 'BlkioDeviceReadIOps': None, 'BlkioDeviceWriteIOps': None, 'CpuPeriod': 0, 'CpuQuota': 0, 'CpuRealtimePeriod': 0, 'CpuRealtimeRuntime': 0, 'CpusetCpus': '', 'CpusetMems': '', 'Devices': [], 'DeviceCgroupRules': None, 'DeviceRequests': None, 'MemoryReservation': 0, 'MemorySwap': 0, 'MemorySwappiness': 0, 'OomKillDisable': False, 'PidsLimit': 2048, 'Ulimits': [{'Name': 'RLIMIT_NOFILE', 'Hard': 524288, 'Soft': 524288}, {'Name': 'RLIMIT_NPROC', 'Hard': 250822, 'Soft': 250822}], 'CpuCount': 0, 'CpuPercent': 0, 'IOMaximumIOps': 0, 'IOMaximumBandwidth': 0, 'MaskedPaths': None, 'ReadonlyPaths': None}
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: 45bcdc1a907c> with image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'> to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/json HTTP/1.1" 200 None
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: 45bcdc1a907c> with image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'> to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/json HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/json HTTP/1.1" 200 None
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: 45bcdc1a907c> with image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'> to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/json HTTP/1.1" 200 None
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: 45bcdc1a907c> with image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'> to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/json HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:41335
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:41335 "GET /notebook/opendatahub/jovyan HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:41335 "GET /notebook/opendatahub/jovyan/lab HTTP/1.1" 200 5694
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495/stop?t=0 HTTP/1.1" 204 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "DELETE /v1.41/containers/45bcdc1a907c0ec6038d542af1882cc8e50ffff6f4a8a71287ef20c305d0c495?v=True&link=False&force=True HTTP/1.1" 204 0
INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/images/quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506/json HTTP/1.1" 200 None
INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
INFO     testcontainers.core.container:container.py:98 Pulling image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'>
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/create HTTP/1.1" 201 88
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/json HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/start HTTP/1.1" 204 0
INFO     testcontainers.core.container:container.py:125 Container started: 04d207776892
INFO     testcontainers.core.docker_client:docker_client.py:66 using host unix:///run/user/16481/podman/podman.sock
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    docker.utils.config:config.py:21 Trying paths: ['/home/wsiqueir/.docker/config.json', '/home/wsiqueir/.dockercfg']
DEBUG    docker.utils.config:config.py:28 No config file found
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /version HTTP/1.1" 200 1025
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/json HTTP/1.1" 200 None
DEBUG    root:workbench_image_test.py:194 {'Binds': [], 'ContainerIDFile': '', 'LogConfig': {'Type': 'journald', 'Config': None}, 'NetworkMode': 'bridge', 'PortBindings': {'8888/tcp': [{'HostIp': '0.0.0.0', 'HostPort': '36941'}]}, 'RestartPolicy': {'Name': 'no', 'MaximumRetryCount': 0}, 'AutoRemove': False, 'VolumeDriver': '', 'VolumesFrom': None, 'ConsoleSize': [0, 0], 'Annotations': {'io.container.manager': 'libpod', 'org.opencontainers.image.stopSignal': '15', 'org.systemd.property.KillSignal': '15', 'org.systemd.property.TimeoutStopUSec': 'uint64 10000000'}, 'CapAdd': [], 'CapDrop': [], 'CgroupnsMode': '', 'Dns': [], 'DnsOptions': [], 'DnsSearch': [], 'ExtraHosts': [], 'GroupAdd': ['0'], 'IpcMode': 'shareable', 'Cgroup': '', 'Links': None, 'OomScoreAdj': 200, 'PidMode': 'private', 'Privileged': False, 'PublishAllPorts': False, 'ReadonlyRootfs': False, 'SecurityOpt': [], 'Tmpfs': {'/opt/app-root/src': 'rw,rprivate,nosuid,nodev'}, 'UTSMode': 'private', 'UsernsMode': '', 'ShmSize': 65536000, 'Runtime': 'oci', 'Isolation': '', 'CpuShares': 0, 'Memory': 0, 'NanoCpus': 0, 'CgroupParent': 'user.slice', 'BlkioWeight': 0, 'BlkioWeightDevice': None, 'BlkioDeviceReadBps': None, 'BlkioDeviceWriteBps': None, 'BlkioDeviceReadIOps': None, 'BlkioDeviceWriteIOps': None, 'CpuPeriod': 0, 'CpuQuota': 0, 'CpuRealtimePeriod': 0, 'CpuRealtimeRuntime': 0, 'CpusetCpus': '', 'CpusetMems': '', 'Devices': [], 'DeviceCgroupRules': None, 'DeviceRequests': None, 'MemoryReservation': 0, 'MemorySwap': 0, 'MemorySwappiness': 0, 'OomKillDisable': False, 'PidsLimit': 2048, 'Ulimits': [{'Name': 'RLIMIT_NOFILE', 'Hard': 524288, 'Soft': 524288}, {'Name': 'RLIMIT_NPROC', 'Hard': 250822, 'Soft': 250822}], 'CpuCount': 0, 'CpuPercent': 0, 'IOMaximumIOps': 0, 'IOMaximumBandwidth': 0, 'MaskedPaths': None, 'ReadonlyPaths': None}
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: 04d207776892> with image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'> to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/json HTTP/1.1" 200 None
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: 04d207776892> with image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'> to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/json HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/json HTTP/1.1" 200 None
INFO     testcontainers.core.waiting_utils:waiting_utils.py:52 Waiting for container <Container: 04d207776892> with image <Image: 'quay.io/wsiqueir/workbench-images:jupyter-minimal-ubi9-python-3.11-texlive_install_with_script_export_on_dockerfile_20250506'> to be ready ...
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/json HTTP/1.1" 200 None
DEBUG    root:docker_utils.py:65 Adding src='/tmp/tmpde3gteg_/test.ipybn' to archive dst='/opt/app-root/src'
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "PUT /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/archive?path=%2Fopt%2Fapp-root%2Fsrc HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/exec HTTP/1.1" 201 74
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/exec/216227cd2581789e4accba63abd9d8668f4072e607b328e9ee083ef81cf35dac/start HTTP/1.1" 101 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "GET /v1.41/exec/216227cd2581789e4accba63abd9d8668f4072e607b328e9ee083ef81cf35dac/json HTTP/1.1" 200 413
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "POST /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6/stop?t=0 HTTP/1.1" 204 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:None "DELETE /v1.41/containers/04d207776892f77a9bae15874e08737d5c79beb66f86ecfc2580f45563a0a3e6?v=True&link=False&force=True HTTP/1.1" 204 0
